README.md

```diff
@@ -71,6 +78,22 @@ format transform `{request>remote_addr} - {request>user_id} [{ts}] "{request>met
 }
 ```
 
+# Alternative value
+
+You can use an alternative value by using the following syntax `{val1:val2}`. For example, to show the `X-Forwarded-For`
+header as `remote_addr` replacement you can do the following
+
+```caddy
+format transform `{request>headers>X-Forwarded-For[0]:request>remote_ip} - {request>user_id} [{ts}] "{request>method} {request>uri} {request>proto}" {status} {size} "{request>headers>Referer>[0]}" "{request>headers>User-Agent>[0]}"` {
+        time_format "02/Jan/2006:15:04:05 -0700"
+}
+```
+
+The character `:` act as indicator for alternative value if the preceding key is not set.
+
+For example, `{request>headers>X-Forwarded-For[0]:request>remote_ip}` means that if `X-Forwarded-For` first array value
+is not empty use that otherwise fallback on `remote_addr`.
```

Comment:
One small fix :) (missing `>` to index the array)

```suggestion
For example, `{request>headers>X-Forwarded-For>[0]:request>remote_ip}` means that if `X-Forwarded-For` first array value is not empty use that otherwise fallback on `remote_addr`.
```